maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -847,12 +847,31 @@ private DefaultScanResult scanDependencies()
         {
             try
             {
+                DefaultScanResult scanResult = new DefaultScanResult( Collections.EMPTY_LIST );
+                
+                List<String> dependenciesToScan = new ArrayList();
```

Comment:
missing Generics on constructor. We have Java 1.6.
`List<String> dependenciesToScan = new ArrayList<String>();`